app/src/main/java/de/berlindroid/zepatch/patchable/Demo.kt

```diff
@@ -88,69 +116,85 @@ fun AndroidLogo() {
 
 @Patch("AndyA")
 @Composable
-fun AndyA() {
-    SafeArea(
-        modifier = Modifier
-            .size(200.dp)
-            .background(
-                brush = Brush.linearGradient(
-                    colors = listOf(
-                        Color.Red,
-                        Color.Yellow,
-                        Color.Green,
-                        Color.Blue,
-                        Color.Cyan
-                    )
-                ),
-                shape = RoundedCornerShape(
-                    topStartPercent = 50,
-                    topEndPercent = 50,
-                    bottomStartPercent = 50,
-                    bottomEndPercent = 0
+fun AndyA(
+    shouldCapture: Boolean = false,
+    onBitmap: (ImageBitmap) -> Unit = {},
+) {
+    var name by remember { mutableStateOf("Andy") }
+    Column {
+        SafeArea(
+            shouldCapture = shouldCapture,
+            onBitmap = onBitmap,
+        ) {
+            Column(
+                modifier = Modifier
+                    .size(200.dp)
+                    .background(
+                        brush = Brush.linearGradient(
+                            colors = listOf(
+                                Color.Red,
+                                Color.Yellow,
+                                Color.Green,
+                                Color.Blue,
+                                Color.Cyan
+                            )
+                        ),
+                        shape = RoundedCornerShape(
+                            topStartPercent = 50,
+                            topEndPercent = 50,
+                            bottomStartPercent = 50,
+                            bottomEndPercent = 0
+                        )
+                    ),
+            ) {
+                Spacer(modifier = Modifier.weight(1f))
+                Text(
+                    modifier = Modifier.fillMaxWidth(),
+                    fontFamily = FontFamily.Cursive,
+                    textAlign = TextAlign.Center,
+                    fontStyle = FontStyle.Italic,
+                    fontWeight = FontWeight.ExtraBold,
+                    fontSize = 70.sp,
+                    text = name,
                 )
-            ),
-    ) {
-        Column(modifier = Modifier.fillMaxSize()) {
-            Spacer(modifier = Modifier.weight(1f))
-            Text(
-                modifier = Modifier.fillMaxWidth(),
-                fontFamily = FontFamily.Cursive,
-                textAlign = TextAlign.Center,
-                fontStyle = FontStyle.Italic,
-                fontWeight = FontWeight.ExtraBold,
-                fontSize = 70.sp,
-                text = "Andy",
-            )
-            Spacer(modifier = Modifier.weight(1f))
+                Spacer(modifier = Modifier.weight(1f))
+            }
         }
+        TextField(value = name, onValueChange = { name = it })
```

Comment:
This makes this particular patch interactive so you can change the name on the patch. The part that will be converted is in the SafeArea. See the screenshot